api/lib/infrastructure/repositories/authentication-method-repository.js

```diff
@@ -63,91 +58,93 @@ module.exports = {
         password: hashedPassword,
         shouldChangePassword: true,
       });
-
       const authenticationMethod = new AuthenticationMethod({
         authenticationComplement,
         identityProvider: AuthenticationMethod.identityProviders.PIX,
         userId,
       });
-
-      const bookshelfAuthenticationMethod = await new BookshelfAuthenticationMethod(authenticationMethod)
-        .save(null, { transacting: domainTransaction.knexTransaction });
-      return _toDomainEntity(bookshelfAuthenticationMethod);
+      const authenticationMethodForDB = _.pick(authenticationMethod, ['identityProvider', 'authenticationComplement', 'externalIdentifier', 'userId']);
+      const knexConn = domainTransaction.knexTransaction || knex;
+      const [authenticationMethodDTO] = await knexConn('authentication-methods').insert(authenticationMethodForDB).returning(COLUMNS);
+      return _toDomain(authenticationMethodDTO);
     } catch (err) {
       if (bookshelfUtils.isUniqConstraintViolated(err)) {
         throw new AlreadyExistingEntityError(`Authentication method PIX already exists for the user ID ${userId}.`);
       }
-      throw err;
     }
   },
 
   async findOneByUserIdAndIdentityProvider({ userId, identityProvider }) {
-    const authenticationMethod = await BookshelfAuthenticationMethod
+    const authenticationMethodDTO = await knex
+      .select(COLUMNS)
+      .from('authentication-methods')
```

Comment:
nitpick: on pourrait mettre le nom de la table qui reviens tout le temps dans une constante
```suggestion
      .from(AUTHENTICATED_METHOD_TABLE)
```